.github/workflows/check-query-files-and-compilation.yml

```diff
@@ -56,15 +56,31 @@ jobs:
         run: |
           bash ./scripts/ci-install-${{ matrix.os }}.sh
 
+      - name: Setup Parsers Cache
+        id: parsers-cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ./parser/
+            ~/AppData/Local/nvim/pack/nvim-treesitter/start/nvim-treesitter/parser/
+          key: ${{ matrix.os }}-parsers-v1-${{ hashFiles('./lockfile.json', './lua/nvim-treesitter/parsers.lua', './lua/nvim-treesitter/install.lua', './lua/nvim-treesitter/shell_selectors.lua') }}
```

Comment:
I think workflows shares cache key, so we have a conflict:

![image](https://user-images.githubusercontent.com/8050659/150589105-7b118064-2622-406d-ae8f-b2940bc151b3.png)

Reply:
yes, also the cache is shared between distinct compilers. It's missing in the cache key and maybe for concurrency also cache nvim tag

Reply:
Those variables should be included in cache key.